splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -327,12 +330,128 @@ def _get_python_version_from_executable(python_binary_name: str) -> str:
         )
 
 
+def summary_report(
+    source: str,
+    ta_name: str,
+    output_directory: str,
+    verbose_report: bool,
+) -> None:
+    # initialising colorama to handle ASCII color in windows cmd
+    c.init()
+    color_palette = {
+        "copied": c.Fore.GREEN,
+        "conflict": c.Fore.RED,
+        "modified": c.Fore.YELLOW,
+    }
+
+    # conflicting files from ucc-gen package folder
+    conflict_path = os.path.join(internal_root_dir, "package")
+    # conflict files generated through-out the process
+    conflict_static_list = frozenset(
+        [
+            "import_declare_test.py",
+            f"{ta_name}_rh_*.py",
+            "app.conf",
+            "inputs.conf*",
+            "restmap.conf",
+            "server.conf",
+            f"{ta_name}_*.conf*",
+            "web.conf",
+            "default.xml",
+            "configuration.xml",
+            "dashboard.xml",
+            "inputs.xml",
+            "openapi.json",
+        ]
+    )
+
+    def line_print(print_path: str, mod_type: str) -> None:
+        if verbose_report:
+            logger.info(
+                color_palette.get(mod_type, "")
+                + str(print_path).ljust(80)
+                + mod_type
+                + c.Style.RESET_ALL,
+            )
+        summary[mod_type] += 1
+
+    def check_for_conflict(file: str, relative_file_path: str) -> bool:
+        conflict_path_file = os.path.join(conflict_path, relative_file_path)
+        if os.path.isfile(conflict_path_file):
+            return True
+        for pattern in conflict_static_list:
+            if fnmatch.fnmatch(file, pattern):
+                return True
+        if file:
```

Comment:
Why do we have this line here?

Reply:
Some leftover code I wanted to use before I switched to pattern method.
 I'll remove it next commit.